### PR TITLE
Add regression test for empty modelcard update

### PR DESCRIPTION
**Pull request by Wauplin**
Fix https://github.com/huggingface/huggingface_hub/issues/1010.

The issue seems to have been solved by @nateraw changes (https://github.com/huggingface/huggingface_hub/pull/940). I added a regression test just in case in the future.

(cc @cakiki)

### CODE REVIEW

File: tests/test_repocard.py [hunk 1]
```diff
@@ -130,6 +130,11 @@
 ---
 """
 
+DUMMY_MODELCARD_NO_TEXT_CONTENT = """---
+license: cc-by-sa-4.0
```
Comment (nateraw) [inline tests/test_repocard.py:134]: would expect this to fail on missing language (but maybe the spec has been updated to not have it be required?)

**Review by nateraw — APPROVED** [review]
Awesome! Thanks for adding this test. Probably good to go (I thought there was conflict originally, but looks like my GH UI wasn't updating haha).

File: tests/test_repocard.py (hunk 1 above)
Comment (Wauplin) [reply tests/test_repocard.py:134]: I don't know to be honest. I took the example from the reported issue to be sure I'm testing what was reported.

**Comment by Wauplin** [comment]
Thanks @nateraw for review. I'm merging.
